extensions/2.0/Khronos/KHR_animation_pointer/README.md

```diff
@@ -0,0 +1,282 @@
+# KHR\_animation_pointer
+
+## Contributors
+
+* Alexey Knyazev
+* Ben Houston, Threekit
+* Bryce Hutchings, Microsoft [@brycehutchings](https://twitter.com/brycehutchings)
+* Don McCurdy
+* Ed Mackey, AGI @emacke
+* Felix Herbst, prefrontal cortex [@hybridherbst](https://twitter.com/hybridherbst)
+* Gary Hsu, Microsoft [@bghgary](https://twitter.com/bghgary)
+* Jamie Marconi, Microsoft [@najadojo](https://twitter.com/najadojo)
+* Joe Herdman, [@jbherdman](https://github.com/jbherdman)
+* Lewis Weaver, Microsoft
+* Marco Hutter
+* Norbert Nopper, UX3D [@UX3DGpuSoftware](https://twitter.com/UX3DGpuSoftware)
+
+Copyright (C) 2018-2022 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension is based on the animation features of glTF 2.0. The structure of the schemas stay the same.  
+  
+The only major addition is, that the output values are mapped using a [JSON Pointer](https://datatracker.ietf.org/doc/html/rfc6901).
+  
+### Motivation
+
+At this point of time, one can only target the transformation or weight data of a node.  
+With this extension, one can technically target any value (`scalar`, `vec2`, `vec3`, `vec4` and scalars in an `array`) in glTF e.g.
+
+* Color factors in materials
+* Camera field of view
+
+Even using a JSON pointer, the targets and their expected behavior are explictly defined.
+
+Current and future extensions have to write against this specification to allow to animate specific values.
```

Comment:
I am afraid of this type of wording in the extension, as much as possible I believe that extensions shall be self contained - this puts a burden on all new extensions to take this extension into consideration.

I would rather prefer that this extension adresses the needed targets for the usecase(s) it sets out to solve.

Reply:
The usecase is being able to animate glTF files beyond the current very strict limitations (colors, UVs, camera FOV, etc. can't be animated).
This includes current and future extensions. Extensions can specify what data is supposed to be animatable.

Reply:
The new animation pointer extension language only defines its JSON structure and compatible accessor types. Actual list of mutable properties is defined in a separate document, which will also be used by interactivity extensions.